README.rst

```diff
@@ -87,6 +87,8 @@ The following parameters are optional:
 - :code:`rp_launch_description = 'Smoke test'` - launch description (could be overridden
   by pytest --rp-launch-description option, default value is '')
 
+- :code:`rp_rerun = True` - treat the Launch as a rerun of the recent Launch of the same name
+- :code:`rp_rerun_of = xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` - treat the Launch as a rerun of the given Launch (to be used together with rp_rerun = True)
```

Comment:
According to the [dev guides](https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/reporting.md#start-launch), `rerun_of` and `rerun` should be used together. So, in our case, it will be enough to only specify `rp_rerun_of` with the launch ID to inform the agent that we want to rerun an existing launch.

Reply:
fair enough.